src/utrition/src/components/imageupload/imageupload.js

```diff
@@ -14,11 +17,81 @@ const ImageUpload = () => {
     upload_type: "image",
   };
 
+  // New header
```

Comment:
Added unique header for backend to handle (let me know if this needs to be changed)